README.md

```diff
@@ -7,3 +7,29 @@ Allow Journal Managers and (optionally) Authors to deposit articles via the SWOR
 This plugin is best installed using the Plugin Gallery from within Open Journal Systems.
 Log in as Administrator, navigate to `Settings` > `Website` > `Plugins`, and open the "Plugin Gallery"
 tab. Find the Sword plugin listed there and use the `Install` button to install it.
+
+## Configuration
+
+### General Settings
+- After successfull installation, you  have to **enable** the plugin in both generic and inmport export plgun section.
+- A new tab, **Sword Settings** will be available next to Website Settigns -> plugins
+- After clicking the tab, ad a new Peposit point using the **Create Deposit Point**
+- Add your Service Provider Information there. 
+  -   Name: Customized name
+  -   Deposit Point URL: Service Document name :   e.g.http://demo.dspace.org/swordv2/servicedocument
```

Comment:
Depending on the deposit point type, the URL will either need to be the service document or the deposit point URL. See the documentation on the form about this.